### PR TITLE
Header

### PR DESCRIPTION
**Pull request by LyriCaGitHubber**
create header area and header stylings

### CODE REVIEW

**Review by phlgr — APPROVED** [review]

File: styles.css [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+  margin: 0;
+}
+
+/* Specific Styles */
+
+header {
+  padding: 10px 0 10px 0;
+  font-family: "Encode Sans SC", sans-serif;
+  text-align: center;
+  width: 100vw;
```
Comment (marwinburesch) [inline styles.css:24]: You can remove this once you've got your grid in place

**Review by mbantwerpes — APPROVED** [review]
Looks good to me. Maybe consider using other units for sizing instead of px, for example 'em' and 'rem'.

File: styles.css [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+
 *,
 *::before,
 *::after {
   box-sizing: border-box;
 }
+/* General Styles */
+h1 {
+  margin: 0;
+  font-size: 50px;
```
Comment (marwinburesch) [inline styles.css:11]: Please check this comment on a different PR: https://github.com/danifosy/fantastic-parakeet/pull/2#discussion_r671065287

**Review by marwinburesch — APPROVED** [review]
Very nice start, excellent focus

File: styles.css (hunk 1 above)
Comment (LyriCaGitHubber) [reply styles.css:24]: good point. thanks :-)

**Pull request merged**